patches/string-length+4.0.2.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/node_modules/string-length/index.js b/node_modules/string-length/index.js
+index c2589a2..3040f0a 100644
+--- a/node_modules/string-length/index.js
++++ b/node_modules/string-length/index.js
+@@ -1,5 +1,5 @@
+ 'use strict';
+-const stripAnsi = require('strip-ansi');
++const stripAnsi = require('strip-ansi-cjs');
```

Comment:
this patch only applies within `eslint-mdx`, will downstream adopters run into an issue if they don't patch too?

Reply:
As above